nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -42,6 +42,8 @@
   Users can use it by `services.displayManager.ly.enable` and config it by
   `services.displayManager.ly.settings` to generate `/etc/ly/config.ini`
 
+- `forgejo` has been upgraded to version 9.0, see the [release notes](https://codeberg.org/forgejo/forgejo/milestone/7235).
```

Comment:
I lean towards moving that release note entry out of the "_Highlights_" section into the "_Backward Incompatibilities_" further down below where we already have two entries for Forgejo.
So all of them are side by side.

Well and jumping from v7 (`forgejo` in `24.05`) to v9 are technically two major updates with breakages.

But let me know what you think -- open to discuss :)

We could also use that opportunity to mention `forgejo-lts` somewhere. Which is something we wanted to do a while ago but postponed so far.

Reply:
I agree it's probably worth moving to the Backward Incompat section.

Maybe something like

> `services.foregejo.package` now defaults to `forgejo-lts`, the Long Term Support version of Forgejo. The latest version can be selected by setting `services.forejo.package = pkgs.forgejo`, with the understanding that this package could have breaking changes during the life of a stable NixOS release.

Reply:
@emilylange 
> I lean towards moving that release note entry out of the "_Highlights_" section into the "_Backward Incompatibilities_" further down below where we already have two entries for Forgejo.
> So all of them are side by side.

sounds like a good idea.

> Well and jumping from v7 (forgejo in 24.05) to v9 are technically two major updates with breakages.

Right, we should mention that as well.

@adamcstephens 
> with the understanding that this package could have breaking changes during the life of a stable NixOS release.

Wouldn't it be better to introduce e.g. `forgejo_9`, so it's a bit more explicit to upgrade? Not sure how to handle this after EOL, just a thought.

Reply:
This was discussed in https://github.com/NixOS/nixpkgs/pull/331187 but there are only ever two releases of forgejo.

Reply:
If a use has changed the default from the LTS version, I personally consider that explicit opting in.

Reply:
If we want to revisit the naming decision again, let's do so in another PR.